sys-apps/pcmciautils/pcmciautils-018_p8-r3.ebuild

```diff
@@ -44,7 +44,6 @@ pkg_setup() {
 		STRIP=true
 		RANLIB="$(tc-getRANLIB)"
 		OPTIMIZATION="${CFLAGS} ${CPPFLAGS}"
```

Comment:
actually: I didn’t notice this earlier but configure arguments really shouldn’t be in pkg_setup anyway!